src/main/java/de/tum/in/www1/artemis/domain/FileUploadSubmission.java

```diff
@@ -25,21 +26,44 @@ public String getSubmissionExerciseType() {
     @Transient
     private final transient FileService fileService = new FileService();
 
-    @Column(name = "file_path")
-    private String filePath;
+    @ElementCollection(fetch = FetchType.EAGER)
+    @CollectionTable(name = "file_submission_paths", joinColumns = @JoinColumn(name = "submission_id"))
```

Comment:
Does this already imply proper cascading to the file paths table? I.e., are the rows connected to a submission deleted when a submission is deleted?

Reply:
Deleting a participation and a submission results in the submission to be removed from the submission table and all it's paths to be removed from the file_submission_paths table.